module/VuFind/src/VuFind/RecordDriver/DefaultRecord.php

```diff
@@ -1210,7 +1210,14 @@ public function getSeries()
      */
     public function getShortTitle()
     {
-        return $this->fields['title_short'] ?? '';
+        if (array_key_exists('title_short', $this->fields)) {
```

Comment:
I'd consider simplifying this method (and the equivalent below) to something like:

```
$titles = (array)($this->fields['title_short'] ?? []);
return $titles[0] ?? '';
```

Pretty sure this has the same outcome in less code.

Reply:
If you think it's too hard to understand, we could introduce a support method like `protected getFirstValueFromField($fieldName)` and put the logic in there... and then the other methods could call it. That would make the individual methods even shorter, and the behavior of the two-line logic would be self-documenting based on the method name.